src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -534,6 +534,30 @@ def prepare_latents(self, batch_size, num_channels_latents, height, width, dtype
         latents = latents * self.scheduler.init_noise_sigma
         return latents
 
+    def enable_freeu(self, s1=0.9, s2=0.2, b1=1.2, b2=1.4):
+        r"""Enables the FreeU mechanism as in https://arxiv.org/abs/2309.11497.
+
+        The default values are for Stable Diffusion v1. They come from the [official
+        repository](https://github.com/ChenyangSi/FreeU).
+
+        The suffxes after the scaling factors represent the stages where they are being applied.
+
+        Please refer to the repository for other combinations of hyperparameter values.
+
+        Args:
+            s1 (`int`): Scaling factor to attenuate the contributions of the skip features.
+            s2 (`int`): Scaling factor to attenuate the contributions of the skip features.
+            b1 (`int`): Scaling factor to amplify the contributions of backbone features.
+            b2 (`int`): Scaling factor to amplify the contributions of backbone features.
```

Comment:
```suggestion
            s1 (`int`): Scaling factor to attenuate the contributions of the skip features.
            s2 (`int`): Scaling factor to attenuate the contributions of the skip features.
            b1 (`int`): Scaling factor to amplify the contributions of backbone features.
            b2 (`int`): Scaling factor to amplify the contributions of backbone features.
```

Can we explain the parameters a bit better here? I.e. `s1` and `s2` should not have the exact same doc strings

Reply:
How about now?